content/blog/announcing-dependency-caching-deployments/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Announcing dependency caching for Pulumi Deployments"
+date: 2024-11-11T18:39:54-05:00
+meta_desc: "Dependency caching for Pulumi Deployments is now available, speeding up your deployments by reusing dependencies across runs."
+meta_image: meta.png
+authors:
+- komal-ali
+- iaroslav-titov
+tags:
+- features
+- pulumi-deployments
+- caching
+---
+
+We’re excited to announce dependency caching for [Pulumi Deployments](/docs/pulumi-cloud/deployments/)! This new feature is designed to significantly speed up deployment times by reducing dependency installation time by up to 80%, ensuring faster and more efficient workflows for every Pulumi user.
+
+When using Pulumi Deployments, installing dependencies can be a time-consuming part of a deployment. With dependency caching, Pulumi can reuse previously installed dependencies, allowing your deployments to skip redundant installation steps and focus on the task at hand. This optimization is especially helpful for frequent deployments, reducing both time and compute resources.
+
+<!--more-->
+
+## How dependency caching works
+
+Pulumi Deployments now automatically caches dependencies for popular package managers, such as npm, pip, and go modules. When a deployment is triggered, Pulumi will:
+
+1.	Check for an existing cache of your dependencies.
+2.	If a cache already exists, Pulumi will reuse those dependencies, bypassing the need to install them from scratch.
+3.	If a cache does not exist, or if dependencies have changed, Pulumi will manage the cache to be used for subsequent deployments.
+
+The end result? Faster, more efficient deployments to push out infrastructure changes quickly.
+
+## How to enable dependency caching
+
+Dependency caching can be enabled through deployment settings, however you choose to configure them.
+
+### In the Pulumi Console
+
+1. Open the Pulumi Console and navigate to a stack's deployment settings (i.e. Stack page > Settings > Deploy).
+2. Under the "Dependency caching" section, toggle the switch to enable dependency caching for your stack.
+3. Click "Save deployment configuration" to apply the changes.
+
+![Dependency caching in the Pulumi Console](enable-caching-console.png)
+
+### Using the REST API
+
+You can also enable dependency caching using the Pulumi REST API. See the [API documentation](/docs/pulumi-cloud/deployments/api/#cache-options) for more information.
```

Comment:
`"You can also ..."` reads to me as the final item in a list of options, but then its followed up by PSP.  Maybe move the REST API item to the end?